ios/RN/RNCamera.h

```diff
@@ -11,7 +11,7 @@
 
 @class RNCamera;
 
-@interface RNCamera : UIView <AVCaptureMetadataOutputObjectsDelegate, AVCaptureFileOutputRecordingDelegate, RNFaceDetectorDelegate>
+@interface RNCamera : UIView <AVCaptureMetadataOutputObjectsDelegate, AVCaptureFileOutputRecordingDelegate, RNFaceDetectorDelegate, AVCaptureVideoDataOutputSampleBufferDelegate>
```

Comment:
As part of the cleanup here, if we aren't using the file output at all, let's remove any orphaned functionality including `AVCaptureFileOutputRecordingDelegate`